app/src/main/res/layout/floating_calculator_basic.xml

```diff
@@ -12,20 +12,23 @@
 
         <Button
             android:id="@+id/digit7"
+            android:tag="seven"
```

Comment:
Maybe change tag value to "digit7", and then rename the audio files to something like "digit_7_en.mp3" and "digit_7_sw.mp3"?